code/modules/reagents/chemistry/reagents.dm

```diff
@@ -112,6 +113,10 @@
 		to_chat(M, "<span class='boldannounce'>You're not feeling good at all! You really need some [name].</span>")
 	return
 
+/datum/reagent/proc/on_reagent_conflict(mob/living/M, datum/reagent/CR)
+	if(!M || !CR)
```

Comment:
on_mob_life depletes the reagent but this doesn't, conflicting reagents never go away. All my math was wrong and if you inject even 0.1u of tricord and 0.1u of omni you will take 4 damage per cycle until you stop metabolizing

can somebody confirm that this is happening or am I losing my mind

merge this please

Reply:
the first check is the reagent so if you deplete either of the reagents this begins returning false.